src/ast.cpp

```diff
@@ -32,7 +33,68 @@ void CreateTableCommand::execute(Database& database)
 
 void InsertIntoCommand::execute(Database& database)
 {
-  std::cout << *this << std::endl;
+  using namespace white::davisbase::sdl;
+  using common::NullValue;
+  using std::holds_alternative;
+
+  auto table_opt = database.getTable(table_name);
+  if (table_opt == std::nullopt) {
+    throw std::runtime_error("Table doesn't exist");
+  }
+
+
+  std::map<std::string, common::ColumnDefinition> columns_map;
+  std::vector<common::ColumnDefinition> column_defs =
+    table_opt.columnDefinitions();
```

Comment:
```
/home/travis/build/ozars/white-davisbase/src/ast.cpp:48:15: error: ‘class std::optional<white::davisbase::sdl::Table>’ has no member named ‘columnDefinitions’
     table_opt.columnDefinitions();
```

table_opt is of type `optional<Table>`. It is necessary to get its value to access `columnDefinitions` method.

```
auto& table = table_opt.value()
auto column_defs = table.columnDefinitions();
```